code/modules/holodeck/computer.dm

```diff
@@ -170,6 +170,8 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 			if(program_to_load)
 				load_program(program_to_load)
 		if("safety")
+			if (!EMAGGED | !issilicon(usr))
```

Comment:
this doesn't work. Bitwise OR is used for bits (if you know how bitwise OR works). What you want is || which is logical OR
EMAGGED is a flag (basically just a number representing a bitfield), you need to compare it to something, e.g. obj_flags
```suggestion
			if (!(obj_flags & EMAGGED) || !issilicon(usr))
```

Reply:
Thank you, for some reason it was working.

Reply:
0 bitwise or 1 is still 1, it's just speaking the wrong language when you read it if that makes sense